src/encoded/static/components/browse/BrowseView.js

```diff
@@ -537,40 +352,102 @@ function BrowseTableWithSelectedFilesCheckboxes(props){
                 }
             })
         });
-
     }, [ columnExtensionMap, selectedFiles, selectFile, unselectFile ]);
 
+    /**
+     * Plain object used for caching to helps to preserve open states of Processed & Raw files' sections to iron out jumping as that state is lost that would
+     * otherwise be encountered during scrolling due to dismounting/destruction of ResultRow components. `detailPaneFileSectionStateCache` is read in constructor
+     * or upon mount by these ResultRows.
+     *
+     * Runs/memoized only once on mount so that same instance of the `detailPaneFileSectionStateCache` object persists throughout lifeycle of
+     * `BrowseTableWithSelectedFilesCheckboxes` component.
+     */
+    const { detailPaneFileSectionStateCache, updateDetailPaneFileSectionStateCache } = useMemo(function(){
+        const detailPaneFileSectionStateCache = {};
+        function updateDetailPaneFileSectionStateCache(resultID, resultPaneState){
+            // Purposely avoid changing reference to avoid re-renders/updates (except when new components initialize)
+            if (resultPaneState === null){
+                delete detailPaneFileSectionStateCache[resultID];
+            } else {
+                detailPaneFileSectionStateCache[resultID] = resultPaneState;
+            }
+        }
+        return { detailPaneFileSectionStateCache, updateDetailPaneFileSectionStateCache };
+    }, []);
+
+    /**
+     * ExperimentSetDetailPane instance will receive props `result`, `containerWidth`, `propsFromTable`, & `rowNumber`
+     * via React.cloneElement in SPC > SearchResultTable.js > ResultRowColumBlock.
+     *
+     * SearchResultTable (& rest of SPC) will soon no longer handle nor pass down selectedFiles so it is passed in (and memoized upon)
+     * here instead.
+     */
+    const detailPane = useMemo(function(){
+        return (
+            <ExperimentSetDetailPane {...{ selectedFiles, selectFile, unselectFile, windowWidth, href, schemas }} paddingWidth={47}
+                initialStateCache={detailPaneFileSectionStateCache} updateFileSectionStateCache={updateDetailPaneFileSectionStateCache} />
+        );
+    }, [ selectedFiles, windowWidth, href, schemas ]);
+
+    /** Some data calculated/derived from search response / `context` for BrowseView UX specifically */
+    const { showClearFiltersButton, countExternalSets, facets } = useMemo(function(){
+        const { filters: contextFilters = null, facets: contextFacets = [] } = context;
```

Comment:
Codacy found an issue: ['context.filters' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6342004)